pywbemcli/_cmd_class.py

```diff
@@ -39,18 +39,20 @@
 #   Common option definitions for class group
 #
 
-# the default is True which is what we want.
+# The default is True which is what we want.
 # TODO: We need to document all these defaults.
 includeclassqualifiers_option = [              # pylint: disable=invalid-name
     click.option('--no-qualifiers', is_flag=True,
                  required=False, default=True,
-                 help='Do not include qualifiers in the response.'
-                      'The default behavior is to include'
-                      'qualifiers in the returned class.')]
+                 help='Do not include qualifiers in the result. '
+                      'The default behavior is to include '
+                      'qualifiers in the returned objects.')]
```

Comment:
Now that I understand the difference between how we default the inclusion of qualifiers in returned classes vs. returned instances, I suggest that we call out "class" and "instance" specifically in the description. I.e. change the description to: "Do not include qualifiers in the returned classes. The default behavior is to include qualifiers in the returned classes."